File: Modul-4/C/PRAK401_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    int kelipatan;
    char simbol;

    printf("Masukkan kelipatan dan simbol : ");
    scanf("%d %c", &kelipatan, &simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {  
            printf("%c ", simbol);
        } else {
            printf("%d ", i);
        }
    }

    return 0;
}

File: Modul-4/Python/PRAK401_2410817210001_MuhammadFadillah.py
kelipatan, simbol = input("Masukkan kelipatan dan simbol : ").split()
kelipatan = int(kelipatan)

hasil = []
for i in range(1, 51):  
    if i % kelipatan == 0:  
        hasil.append(simbol)
    else:
        hasil.append(str(i))

print(" ".join(hasil))